### ISSUE TITLE
Would you accept a PR for "opacity"?

### ISSUE THREAD

**Issue by Kazade** [libRocket/libRocket#258]
In my local copy of libRocket, I've implemented a basic "opacity" property (it doesn't work with images, but it does with most elements). You can see the commit in question here: https://github.com/Kazade/KGLT/commit/2d24d35a1540d40d9d576e664c5d38bfcb45ba28

Before I clean it up into a PR, I just wanted to find out if such a patch would be accepted?

**Comment by chrismdp** [comment]
LGTM (although I have no power to add stuff to libRocket)

I'd just set it up as a PR, personally. Someone is more likely to just merge it in - it's easier for contributors/owners to deal with.

**Comment by chrismdp** [comment]
...have you also thought about adding the opacity support to the built-in image decorators? Not sure they would be covered by your change?

**Comment by dwimsey** [comment]
Heh, for some reason this slipped past me.

I certainly have no objections at first glance.  I would be happier if it contained an example, but thats not a requirement, just trying to move that direction in general for new features.

You usage works like CSS3, so thats really the defining factor for me as far as the feature is concerned.  The existing code looks ok from a visual inspection, it'd need to be turned in to a PR so my build farm will pick it up and try it on all the various platforms.  It can't break any of the example apps (no reason why it should from what I see) or build in general.

Long story short, yes, it'd probably get accepted, just may not be quickly due to my day job.  This one is pretty simple so it'd be faster than most :)

**Comment by Kazade** [comment]
https://github.com/libRocket/libRocket/pull/262
